<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatershedMarkerDialog</class>
 <widget class="QDialog" name="WatershedMarkerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Watershed Marker</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="11">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="11">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>10</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>614</width>
        <height>313</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="scrollAreaHorizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QImageInteractiveDrawer" name="drawer" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="10" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="previewButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>(Space)</string>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
     <property name="icon">
      <iconset theme="document-preview">
       <normaloff/>
      </iconset>
     </property>
     <property name="shortcut">
      <string>Space</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignLeft|Qt::AlignBottom">
    <widget class="QLabel" name="zoomLabel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="10" alignment="Qt::AlignRight|Qt::AlignBottom">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="resetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="icon">
      <iconset theme="document-revert">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="undoButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>(Backspace)</string>
     </property>
     <property name="text">
      <string>Undo</string>
     </property>
     <property name="icon">
      <iconset theme="edit-undo">
       <normaloff/>
      </iconset>
     </property>
     <property name="shortcut">
      <string>Backspace</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="redoButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Redo</string>
     </property>
     <property name="icon">
      <iconset theme="edit-redo">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="autoButton">
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="icon">
      <iconset theme="roll">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="4" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="brushModeButton">
     <property name="text">
      <string>Marker</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" colspan="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QImageInteractiveDrawer</class>
   <extends>QWidget</extends>
   <header>qimageinteractivedrawer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WatershedMarkerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WatershedMarkerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
